src/components/dialog-full-screen/dialog-full-screen.pw.tsx

```diff
@@ -0,0 +1,565 @@
+  );
+  await expect(closeIcon).toHaveCSS("outline", "rgba(0, 0, 0, 0) solid 3px");
+};
+
+test.describe("render DialogFullScreen component and check properties", () => {
+  test(`should close component after click on closeIcon`, async ({
+    mount,
+    page,
+  }) => {
+    await mount(<DialogFullScreenComponent showCloseIcon />);
+
+    const closeIcon = getDataElementByValue(page, "close").first();
+    const dialogFullScreen = page.getByRole("dialog");
+    await expect(closeIcon).toBeAttached;
+    await expect(dialogFullScreen).toBeAttached;
+    await closeIcon.click();
+    await expect(dialogFullScreen).not.toBeAttached;
+  });
+
+  specialCharacters.forEach((title) => {
+    test(`should render component using ${title} as title`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<DialogFullScreenComponent title={title} />);
+
+      await expect(getDataElementByValue(page, "title")).toHaveText(title);
+    });
+  });
+
+  specialCharacters.forEach((subtitle) => {
+    test(`should render using ${subtitle} as subtitle`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(
+        <DialogFullScreenComponent
+          title="Dialog Full Screen Title"
+          subtitle={subtitle}
+        />
+      );
+
+      await expect(getDataElementByValue(page, "subtitle")).toHaveText(
+        subtitle
+      );
+    });
+  });
+
+  specialCharacters.forEach((childrenValue) => {
+    test(`should render component with ${childrenValue} as a children`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(
+        <DialogFullScreenComponent>{childrenValue}</DialogFullScreenComponent>
+      );
+
+      await expect(getDataElementByValue(page, "form-content")).toContainText(
+        childrenValue
+      );
+    });
+  });
+
+  test("should render with disabledEscKey prop and not be closed after clicking Escape button", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<DialogFullScreenComponent disableEscKey />);
+
+    const dialogFullScreen = page.getByRole("dialog");
+    await expect(dialogFullScreen).toBeAttached;
+    await page.keyboard.press("Escape");
+    await expect(dialogFullScreen).not.toBeAttached;
+  });
+
+  test("should close after pressing Escape button", async ({ mount, page }) => {
+    await mount(<DialogFullScreenComponent />);
+
+    const dialogFullScreen = page.getByRole("dialog");
+    await expect(dialogFullScreen).toBeAttached;
+    await page.keyboard.press("Escape");
+    await expect(dialogFullScreen).not.toBeAttached;
+  });
+
+  test("should call the cancel action after closing", async ({
+    mount,
+    page,
+  }) => {
+    let called = false;
+    await mount(
+      <DialogFullScreenComponent
+        onCancel={() => {
+          called = true;
+        }}
+      />
+    );
+
+    const closeIcon = getDataElementByValue(page, "close").first();
+    await closeIcon.click();
+    expect(called).toBeTruthy();
```

Comment:
I suppose we are trying to see if the returned value for called is true/false, so can this be written as "expect(called).toBe(true);"